docs/getting-started.md

```diff
@@ -310,13 +320,13 @@ You will need to adjust the `img` and `lua` paths according to their location, a
 
 You might also want to add a simple one-line script file to your `~/bin` directory to wrap this command up.
 
-[↑ back to matrix](#task-os-selector)
+[↑ back to matrix](#task--os-selector)
 
 ## Upload LFS image
 
 The compiled LFS image file (e.g. `lfs.img`) is uploaded as a regular file to the device file system (SPIFFS). You do this just like with Lua files with e.g. [ESPlorer](#esplorer) or [NodeMCU-Tool](#nodemcu-tool). There is also a new example, [HTTP_OTA.lua](https://github.com/nodemcu/nodemcu-firmware/tree/dev/lua_examples/lfs/HTTP_OTA.lua), in `lua_examples` that can retrieve images from a standard web service.
 
-Once the LFS image file is on SPIFFS, you can execute the [node.flashreload()](../modules/node/#nodeflashreload) command and the loader will then load it into flash and immediately restart the ESP module with the new LFS loaded, if the image file is valid. However, the call will return with an error _if_ the file is found to be invalid, so your reflash code should include logic to handle such an error return.
+Once the LFS image file is on SPIFFS, you can execute the [node.flashreload()](modules/node.md#nodeflashreload) command and the loader will then load it into flash and immediately restart the ESP module with the new LFS loaded, if the image file is valid. However, the call will return with an error _if_ the file is found to be invalid, so your reflash code should include logic to handle such an error return.
```

Comment:
The `.md` part will fail on RTD as it's https://nodemcu.readthedocs.io/en/latest/modules/node/#nodeflashreload.

Reply:
the link "(build.md#build-options)" in the same document works on RTD whereas (../modules/node/#nodeflashreload) does not (and not only because of the wrong anchor)

Reply:
Oh, nevermind. It really has been too long since I last did serious work with our docs. You are absolutely right. While generating HTML MkDocs drops the `.md` filename extension to build cross-references.